src/test/java/io/castle/client/CastleTest.java

```diff
@@ -0,0 +1,96 @@
+package io.castle.client;
+
+import io.castle.client.internal.backend.RestApiFactory;
+import io.castle.client.internal.config.CastleConfiguration;
+import io.castle.client.model.CastleSdkConfigurationException;
+import org.assertj.core.api.Assertions;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+public class CastleTest {
+
+    @Test
+    public void sdkVerificationPassBecauseOfClassPathConfiguration() throws CastleSdkConfigurationException {
+        //Given
+
+        //When
+        Castle.verifySdkConfigurationAndInitialize();
+        //Then
+        // No exception is throw, because the tests classpath contains the file castle_sdk.properties
+    }
+
+    @Test
+    public void sdkOnTestLoadTheCorrectConfiguration() throws CastleSdkConfigurationException {
+        //Given
+        Castle.verifySdkConfigurationAndInitialize();
+
+        //When the sdk is initiated
+        Castle sdk = Castle.sdk();
+
+        //Then the configuration match the expected values from the class path properties
+        CastleConfiguration sdkConfiguration = sdk.getSdkConfiguration();
+        Assertions.assertThat(sdkConfiguration)
+                .extracting("apiSecret", "castleAppId")
+                .containsExactly("test_api_secret", "test_app_id");
+
+        Assertions.assertThat(sdkConfiguration.getBlackListHeaders())
+                .containsExactlyInAnyOrder(
+                        "cookie",
+                        "testblack"
+                );
+        Assertions.assertThat(sdkConfiguration.getWhiteListHeaders())
+                .containsExactlyInAnyOrder(
+                        "testwhite",
```

Comment:
where are the values `testwhite` and `testblack` coming from?

Reply:
Properties are loaded from the default configuration file in test/resources/castle_sdk.properties